src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -77,6 +77,7 @@
     <PackageReference Include="NuGet.ProjectModel" Version="4.6.0" />
     <PackageReference Include="NuGet.Protocol" Version="4.6.0" />
     <PackageReference Include="NuGet.Versioning" Version="4.6.0" />
+    <PackageReference Include="protobuf-net" Version="2.4.4" />
```

Comment:
Is this needed in this project? Would some csproj in java.interop get this assembly to the output folder?

Reply:
Nah, it wasn't the fix I was looking for.  It should get the assembly because it is referenced by `Xamarin.Android.Tools.Bytecode`.